packages/smooth_app/lib/l10n/app_en.arb

```diff
@@ -1,54 +1 @@
-{
```

Comment:
He's skipping app_en.arg because it's not translated.

Reply:
?

The PR is not skipping the file, it's emptying it.

Reply:
because it's not a template, and it's not translatable on Crowdin, because we disabled English ?

Reply:
maybe because there's no upload_language set in the workflow options, can you try this?
https://github.com/openfoodfacts/smooth-app/pull/59